tests/test_api_handler.py

```diff
@@ -0,0 +1,29 @@
+import unittest
+import asyncio
+from aiounittest import async_test
+from Modules.api_handler import *
+
+
+class APIHandlerTest(unittest.TestCase):
+    handler : BaseWebsocketAPIHandler
+
+    def setUp(self):
+        handler = BaseWebsocketAPIHandler(hostname="dev.api.fuelrats.com", tls=True)
```

Comment:
I don't think this will run. You'll have to do `self.handler = BaseWebsocketAPIHandler(hostname="dev.api.fuelrats.com", tls=True)` instead and remove the class-level type hint